eclipse-theia/theia
# Add localization for empty navigator

#### What it does

Removes a long-standing `TODO` about localizing the message for empty multi-root workspaces.

#### How to test

1. Create an empty workspace file:
```json
{
  "folders": [],
  "settings": {}
}
```
2. Open the workspace.
3. Observe that the message in the widget is correctly localized

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
